nntrainer/utils/parse_util.cpp

```diff
@@ -278,14 +278,15 @@ unsigned int parseType(std::string ll, InputType t) {
  * gamma_initializer = 23
  * beta_initializer = 24
  * input_layers = 25
+ * output_layers = 26
```

Comment:
something strange about this count - its 27 in Layer::PropertyType
please check
I think modelfile is missing here in docs.